python/cuml/test/test_make_classification.py

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pytest
+from cuml.datasets.classification import make_classification
+
+
+@pytest.mark.parametrize('n_samples', [1000])
+@pytest.mark.parametrize('n_features', [100])
+@pytest.mark.parametrize('n_classes', [2, 4])
```

Comment:
It would be nice if we could have a test that actually demonstrates the covariance between the features for different n_informative. `make_blobs` is a tad easier since it's univariate and isotropic, but we might consider lifting a few tests from sklearn: https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/datasets/tests/test_samples_generator.py#L64.

(I also think we should do this w/ make blobs when you get to that point).